src/app/resperson/delete/get.controller.js

```diff
@@ -0,0 +1,32 @@
+const CookieModel = require('../../../common/models/Cookie.class');
+const logger = require('../../../common/utils/logger')(__filename);
+const resPersonApi = require('../../../common/services/resPersonApi');
+
+module.exports = (req, res) => {
+  const cookie = new CookieModel(req);
+  const errMsg = { message: 'Failed to delete responsible person. Try again' };
+  const responsiblePersonId = req.session.deleteResponsiblePersonId;
+  logger.debug('In responsible person / delete get controller');
+
+  if (responsiblePersonId === undefined) {
+    res.redirect('/resperson');
+    return;
+  }
+
+  resPersonApi.deleteResponsiblePerson(cookie.getUserDbId(), responsiblePersonId)
+    .then((apiResponse) => {
+      const parsedResponse = JSON.parse(apiResponse);
+      if (Object.prototype.hasOwnProperty.call(parsedResponse, 'message')) {
+        req.session.errMsg = errMsg;
```

Comment:
No log here if an error occurs, may make it hard to debug in the future.

Reply:
log added.

Reply:
This is good.